### DISCUSSION TITLE
Error With Closing Ticket

### DISCUSSION THREAD

**Discussion by Ignelis91** [LunarTaku/global-ticket-system-v2#2] [Q&A]
![image](https://user-images.githubusercontent.com/110096365/189385125-bc06c06f-041f-4d35-a6c6-d34376f01529.png)
Got This Error Closing Ticket

**Comment by LunarTaku** [comment]
Try and create a new ticket and delete it, if the same error occurs the bot doesn't have permissions to change channel perms

**Reply by Ignelis91** [reply]
Still Same Bot Has All Permissions

**Reply by LunarTaku** [reply]
Did you check if the ticket Id is being saved to the database?

**Reply by Ignelis91** [reply]
yes

**Reply by LunarTaku** [reply]
did you change any code?

**Reply by Ignelis91** [reply]
just language

**Reply by LunarTaku** [reply]
hmm there might be an error with the server or your bot cause this has never happened to me, try using your bot in another server
